tasks/UsblDock.cpp

```diff
@@ -29,14 +29,11 @@ bool UsblDock::configureHook()
     if (! UsblDockBase::configureHook())
         return false;
 
-    if (!_io_port.get().empty())
+    if(_interface.get() != ETHERNET)
```

Comment:
I would actually have picked the other solution ... do not check, as this currently makes the package completely flatfish-specific.